src/Akeneo/Platform/Bundle/ImportExportBundle/Infrastructure/Step/UploadStep.php

```diff
@@ -52,7 +52,7 @@ protected function doExecute(StepExecution $stepExecution)
             throw new \LogicException('malformed job parameters, missing storage configuration');
         }
 
-        if ('local' === $jobParameters[self::STORAGE_KEY]['type'] || 'none' === $jobParameters[self::STORAGE_KEY]['type']) {
+        if ('none' === $jobParameters[self::STORAGE_KEY]['type']) {
```

Comment:
If we don't remove it
```suggestion
        if (NoneStorage::type === $jobParameters[self::STORAGE_KEY]['type']) {
```